### DISCUSSION TITLE
Row drag in grouped state

### DISCUSSION THREAD

**Discussion by MrFahrenheit-55** [bosskmk/pluto_grid#840] [General]
Hi

Is there any progress on getting row drag enabled when the rows have been grouped? I need to be able to reorder the lowest level of the nested tree view without being able to drag the children into a new parent. 

![image](https://github.com/bosskmk/pluto_grid/assets/83400730/4ce5a302-b7b5-42d0-9030-7e47a34cadcc)

Thanks for the great package so far.

**Comment by bosskmk** [comment]
I was trying to add this feature.

The link below is a blog that recorded the parts I was worried about when adding this.
https://pluto.weblaze.dev/plutogrid-development-news-row-grouping#heading-development-drag-and-move-rows-in-a-grouped-state

In summary, depending on the state of the grouped rows, rows should be deleted, created or moved. (In case of multiple selection of multiple rows with different parents or hierarchies)

This can cause users to be confused about which rows will be moved, deleted or created.

I plan to apply this again if there is a good idea that can be expressed in a user-friendly UI.
